R/default.R

```diff
@@ -273,6 +273,12 @@ gsynth.default <- function(formula = NULL,data, # a data frame (long-form)
         cat("\nFor clustered bootsrap, please use the nonparametric procedure.\n")
     }
 
```

Comment:
These lines check that the number of treated units is greater than 40 if the nonparametric bootstrap has been selected. Sorry about all the other edits - my editor automatically removes trailing whitespaces.